tests/backup/backup_helper.go

```diff
@@ -2887,3 +2891,61 @@ func DeleteAppNamespace(namespace string) error {
 	}
 	return nil
 }
+
+// RegisterCluster adds the cluster with the given name
+func RegisterCluster(orgID string, clusterName string, cloudCredName string, ctx context.Context) error {
```

Comment:
nit: Please change the order of the inputs - `clusterName string, cloudCredName string, orgID string, ctx context.Context`